app/core/Multichain/test/utils.test.ts

```diff
@@ -87,6 +91,22 @@ const mockBTCAccount: InternalAccount = {
   type: BtcAccountType.P2wpkh,
 };
 
+const mockSolAccount: InternalAccount = {
+  address: SOL_ADDRESSES,
+  id: '1',
+  type: SolAccountType.DataAccount,
+  methods: [SolMethod.SendAndConfirmTransaction],
+  options: {},
+  metadata: {
+    name: 'Solana Account',
+    importTime: 1684232000456,
+    keyring: {
+      type: KeyringTypes.hd,
```

Comment:
This should be attached to the Snap keyring:

```suggestion
      type: KeyringTypes.snap,
```

But I realized we also made the same mistake for BTC 🙈 If we change that, we might wanna add a dummy `snap` metadata as well (but looks like it's not truly needed for now and we don't have it either on the extension 🙃).

Reply:
I will add both to this account and the BTC account.